docs/configuration/configuration.md

```diff
@@ -451,6 +451,26 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Lower limit for the growth factor of one bucket to the next in each native histogram. 
+# The resolution of a histogram with a lower growth factor will be reduced until it is within the limit.
+# To set an upper limit for the schema (equivalent to "scale" in OTel's exponential histograms), use the following factor limits:
```

Comment:
Let's better break this lines so that an 80 character line length is not exceeded. 
These lines are rendered as they are formatted here. See how it looks on the website: https://prometheus.io/docs/prometheus/latest/configuration/configuration